src/containers/Authentication/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import Register from "../../components/Register";
+import Login from "../../components/Login";
+import { Form, Row, Button } from "antd";
+import "./index.css";
+
+const layout = {
```

Comment:
I see these are used alot maybe we can move it to another file and import it from there instead of writing it everytime in the component ?